2-mandatory/3-stocks.js

```diff
@@ -47,10 +61,28 @@ function getAveragePrices(closingPricesForAllStocks) {
                 (Apple's price on the 5th day) - (Apple's price on the 1st day) = 172.99 - 179.19 = -6.2
     The price change value should be rounded to 2 decimal places, and should be a number (not a string)
 */
-function getPriceChanges(closingPricesForAllStocks) {
-    // TODO
+
+
+function getPriceChanges(array) {
+
+    let changeInPrice = [ ];
+
+    for (i = 0; i < array.length; i++) {
```

Comment:
Issue: The iterator variable `i` in a for loop should be declared with `let`.
Suggestion: Use a for...of loop instead of a for loop to improve readability